server/src/com/treode/server/Schema.scala

```diff
@@ -5,6 +5,8 @@ import com.treode.twitter.finagle.http.BadRequestException
 import scala.collection.mutable.HashMap
 
 class Schema (mapping: HashMap [String, Long]) {
+  
+  def map: HashMap [String, Long] = mapping 
```

Comment:
You could instead declare this as

```
class Schema (val map: Map [String, Long])
```

Two things have happened there: 1) it can be accessed as a property, 2) it's immutable.